python/effects/stop_the_light.py

```diff
@@ -26,7 +26,8 @@
 BLACK = (0, 0, 0)
 
 class StopTheLight(Effect):
-    def __init__(self, strip_bottom=0, strip_top=15):
+    def __init__(self, layout, n_pixels, strip_bottom=5, strip_top=20):
+        Effect.__init__(self, layout, n_pixels)
```

Comment:
Thank you for not using super! super in __init__ is frowned upon: https://fuhm.net/super-harmful/